int32 id
int32 cls
float32 score
BoundingBox bbox
float32 dist_m
float32 speed_mps
float32 speed_kph
int32 num_radar_points
